File: BusinessLogic/GrpcServiceSep3-master/GrpcService/Model/Group.cs
using System;

namespace GrpcService.Model
{
    public class Group
    {
        /// <summary>
        /// Instance variables with get and set
        /// </summary>
        public int id { get; set; }
        public String name { get; set; }

        /// <summary>
        /// 2-argument constructor
        /// </summary>
        /// <param name="id"></param>
        /// <param name="name"></param>
        public Group(int id, string name)
        {
            this.id = id;
            this.name = name;
        }
    }
}

File: PersistenceServer/Sep3JavaServer-master/src/main/java/Model/Group.java
package Model;

public class Group {

    /**
     * Instance variables
     */
    private int id;
    private String name;

    /**
     * Constructor for group
     * @param id
     * @param name
     */
    public Group(int id, String name) {
        this.id = id;
        this.name = name;
    }

    /**
     * Get id
     * @return id
     */
    public int getId() {
        return id;
    }

    /**
     * Set id
     * @param id
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * Get name
     * @return name
     */
    public String getName() {
        return name;
    }

    /**
     * Set name
     * @param name
     */
    public void setName(String name) {
        this.name = name;
    }
}
